### DISCUSSION TITLE
Fluent Generics in C#

### DISCUSSION THREAD

**Discussion by giscus** [Tyrrrz/Tyrrrzme#27] [Blog]
# Fluent Generics in C#

Generic programming is a powerful feature available in many statically typed languages. It offers a way to write code that seamlessly operates against many different types, by targeting the features they share rather than the types themselves. This…

https://tyrrrz.me/blog/fluent-generics

**Comment by jakubsuchybio** [comment]
Finally got to reading it after 3 years and I have to say that I never seen this approach before, but I like it.
